app/src/main/java/me/ccrama/redditslide/ColorPreferences.java

```diff
@@ -598,92 +598,92 @@ public int getColor(String s) {
-        amoled_indigo(R.style.indigo_AMOLED, "amoled_indigo", R.color.md_indigo_A700, 2),
-        blue_indigo(R.style.indigo_blue, "blue_indigo", R.color.md_indigo_A700, 3),
-        amoled_light_indigo(R.style.indigo_AMOLED_lighter, "amoled_light_indigo", R.color.md_indigo_A700, 4),
+        dark_deeporange(R.style.deeporange_dark, "dark_deeporange", R.color.md_deep_orange_A400, 0),
+        light_deeporange(R.style.deeporange_LIGHT, "light_deeporange", R.color.md_deep_orange_A400, 1),
+        amoled_deeporange(R.style.deeporange_AMOLED, "amoled_deeporange", R.color.md_deep_orange_A400, 2),
+        blue_deeporange(R.style.deeporange_blue, "blue_deeporange", R.color.md_deep_orange_A400, 3),
+        amoled_light_deeporange(R.style.deeporange_AMOLED_lighter, "amoled_light_deeporange", R.color.md_deep_orange_A400, 4),
+
+        dark_amber(R.style.amber_dark, "dark_amber", R.color.md_amber_A400, 0),
+        light_amber(R.style.amber_LIGHT, "light_amber", R.color.md_amber_A400, 1),
+        amoled_amber(R.style.amber_AMOLED, "amoled_amber", R.color.md_amber_A400, 2),
+        blue_amber(R.style.amber_blue, "blue_amber", R.color.md_amber_A400, 3),
+        amoled_light_amber(R.style.amber_AMOLED_lighter, "amoled_light_amber", R.color.md_amber_A400, 4),
+
+        dark_yellow(R.style.yellow_dark, "dark_yellow", R.color.md_yellow_A400, 0),
+        light_yellow(R.style.yellow_LIGHT, "light_yellow", R.color.md_yellow_A400, 1),
+        amoled_yellow(R.style.yellow_AMOLED, "amoled_yellow", R.color.md_yellow_A400, 2),
+        blue_yellow(R.style.yellow_blue, "blue_yellow", R.color.md_yellow_A400, 3),
+        amoled_light_yellow(R.style.yellow_AMOLED_lighter, "amoled_light_yellow", R.color.md_yellow_A400, 4),
+
+        dark_lime(R.style.lime_dark, "dark_lime", R.color.md_lime_A400, 0),
+        light_lime(R.style.lime_LIGHT, "light_lime", R.color.md_lime_A400, 1),
+        amoled_lime(R.style.lime_AMOLED, "amoled_lime", R.color.md_lime_A400, 2),
+        blue_lime(R.style.lime_blue, "blue_lime", R.color.md_lime_A400, 3),
+        amoled_light_lime(R.style.lime_AMOLED_lighter, "amoled_light_lime", R.color.md_lime_A400, 4),
+
+        dark_green(R.style.green_dark, "dark_green", R.color.md_green_A400, 0),
+        light_green(R.style.green_LIGHT, "light_green", R.color.md_green_A400, 1),
+        amoled_green(R.style.green_AMOLED, "amoled_green", R.color.md_green_A400, 2),
+        blue_green(R.style.green_blue, "blue_green", R.color.md_green_A400, 3),
+        amoled_light_green(R.style.green_AMOLED_lighter, "amoled_light_green", R.color.md_green_A400, 4),
+
+        dark_teal(R.style.teal_dark, "dark_teal", R.color.md_teal_A700, 0),
+        light_teal(R.style.teal_light, "light_teal", R.color.md_teal_A700, 1),
+        amoled_teal(R.style.teal_amoled, "amoled_teal", R.color.md_teal_A700, 2),
+        blue_teal(R.style.teal_blue, "blue_teal", R.color.md_teal_A700, 3),
```

Comment:
It doesn't needs to. I think the palette is open enough to have options. I chose teal A700, because is a bit darker than A400, and since I'm almost sure you won't be tinting fab icons according to fab color brightness, I decided to use it so white icon is more visible. I was going to do the same with yellow, but yellow a700 looked similar to amber a400 so I stayed with a400.